addons/website_sale/views/templates.xml

```diff
@@ -588,9 +596,15 @@
         </xpath>
     </template>
 
+    <template id="recently_viewed_products_product" inherit_id="website_sale.product" active="False" customize_show="True" name="Recently Viewed Products">
+        <xpath expr="//div[@t-field='product.website_description']" position="after">
+            <t t-call="website_sale.s_products_recently_viewed"/>
+        </xpath>
+    </template>
+
     <!-- Product options: OpenChatter -->
     <template id="product_comment" inherit_id="website_sale.product" active="False" customize_show="True" name="Discussion and Rating">
-        <xpath expr="//div[@t-field='product.website_description']" position="after">
+        <xpath expr="//div[@id='open_chatter']" position="replace">
```

Comment:
replace?

Reply:
The point is to have the chatter at the end so recently viewed products are above it. since the div open_chatter is there only for that I just replace it.